instrumentation/async-http-client/async-http-client-2.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/asynchttpclient/v2_0/AsyncHttpClientSingletons.java

```diff
@@ -23,23 +21,19 @@
   private static final Instrumenter<RequestContext, Response> INSTRUMENTER;
 
   static {
-    HttpClientAttributesExtractor<RequestContext, Response> httpAttributesExtractor =
-        new AsyncHttpClientHttpAttributesExtractor();
-    SpanNameExtractor<RequestContext> spanNameExtractor =
-        HttpSpanNameExtractor.create(httpAttributesExtractor);
-    SpanStatusExtractor<RequestContext, Response> spanStatusExtractor =
-        HttpSpanStatusExtractor.create(httpAttributesExtractor);
+    AsyncHttpClientHttpAttributesGetter httpAttributesExtractor =
+        new AsyncHttpClientHttpAttributesGetter();
```

Comment:
`httpAttributesExtractor` -> `httpAttributesGetter`

there are several of these, worth a search for `Getter [a-zA-Z]*Extractor`

Reply:
doh, we probably duplicated efforts. 🤷🏻 coverage I guess. heh.